doc/Completion.md

```diff
@@ -0,0 +1,58 @@
+﻿# WinGet Command Line Tab Completion
+
+WinGet offers a `complete` command that can be leveraged by your shell to provide context sensitive tab completion. It allows for completion of command names, argument names, and argument values, dependent on the current command line state.
+
+> Note, this feature has not yet been released. Hold tight, it is coming soon!
```

Comment:
Are you going to be able to remember this line when it's released?

Reply:
That's the plan, but if not I'm sure someone will poke us.